packages/gatsby/src/commands/develop-process.ts

```diff
@@ -151,6 +151,7 @@ module.exports = async (program: IDevelopArgs): Promise<void> => {
     parentSpan,
     app,
     pendingQueryRuns: new Set([`/`]),
+    isFirstRun: true,
```

Comment:
Consider naming it to be specific to type gen. It's being reset in some place relevant to typgen and I think keeping `isFirstRun` name could encourage usage of it elsewhere and then place where it is reset might become important / source of weird bugs